R/build.r

```diff
@@ -264,13 +264,14 @@
 #' }
 build_site <- function(pkg = ".",
                        examples = TRUE,
-                       document = FALSE,
+                       document = devel,
```

Comment:
I wonder if it would be better to deprecate `document`? I don't think setting `document` and `devel` independently is important.

Reply:
Yes I agree. Should I just remove it or add a soft deprecation notice?

Reply:
Do you mind following https://github.com/tidyverse/principles/issues/67 ? (Or commenting where you think that advice is bad)